src/main/java/carpet/script/api/Scoreboards.java

```diff
@@ -396,6 +402,132 @@ public static void apply(Expression expression)
             }
             return LazyValue.TRUE;
         });
+
+        expression.addLazyFunction("bossbar", -1, (c, t, lv) ->
+        {
+            BossBarManager bossBarManager = ((CarpetContext)c).s.getMinecraftServer().getBossBarManager();
+            if(lv.size() > 3) throw new InternalExpressionException("'bossbar' accepts max three arguments");
+            if(lv.size() == 0) {
+                Value ret = ListValue.wrap(bossBarManager.getAll().stream().map(CommandBossBar::getId).map(Identifier::toString).map(StringValue::of).collect(Collectors.toList()));
+                return (_c, _t) -> ret;
+            } else if(lv.size() == 1) {
```

Comment:
since you have a return in the previous id statement, this can be replaced with `if`, not `else if`
This will make so that getting the identifier for a bossbar from this else if and next else if can be extracted and shared.